### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Application_Screenreader : All the 'Tab' controls are announced with 'selected' state for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3502]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG1.3.1;#WCAG4.2.1;#WCAG1.3.3;#MAC;#A11ySev2;#Benchmark;#Screenreader;#VoiceOver;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Start screen reader and navigate through the tab controls such as "Application", etc.
4. Observe if the proper 'selected' state is announced or not. 

### Actual:
When screen reader users navigate through the tab controls such as "Application", "Data Explorers" etc., for all the controls it is announced with state as 'selected' which is incorrect. Visually only one of the Tab control whose name is displayed in bold style is the currently selected control.

This issue is observed in the below scenarios:
1. Settings - Data Explorers
2. Settings - Transfers

### Expected:
For screen reader users not all Tab controls present on the screen should be announced with the 'selected' state only that control that is currently selected visually should be announced with 'selected' state.

### User Impact:
If proper information is not conveyed to the screen reader users regarding the controls present on the screen then the users will not know how to interact with those controls and this will lead to confusion for the screen reader users.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EXTdo6oL6JBDhBsxXlJy8ooBhE6RbIZ4YedqB61EXE7YEg?e=6F8LhC

**Comment by JasonYeMSFT** [comment]
@MayuriNaik Could you please verify and close this issue if it is fixed?

**Comment by MayuriNaik** [comment]
For screen reader users, only the currently selected tab is announced with 'selected' sate while navigation. The bug is fixed and hence closing the bug. Verified in application version Version: 1.16.0-dev AzCopy Version: 10.6.0

**Comment by Ashutoshsahoo164** [comment]
GitHubTags : #Closed
